docs/intro/index.txt

```diff
@@ -32,10 +32,10 @@ place: read this material to quickly get up and running.
     `list of Python resources for non-programmers`_
 
     If you already know a few other languages and want to get up to speed with
-    Python quickly, we recommend `Dive Into Python`_. If that's not quite your
+    Python quickly, we recommend `Python documentation`_. If that's not quite your
     style, there are many other `books about Python`_.
 
     .. _python: https://www.python.org/
     .. _list of Python resources for non-programmers: https://wiki.python.org/moin/BeginnersGuide/NonProgrammers
-    .. _Dive Into Python: https://diveinto.org/python3/table-of-contents.html
+    .. _Python documentation: https://docs.python.org/3/
```

Comment:
I would leave the `/3/` out and point to the docs' root, in case this changes at some point in the future:
```suggestion
    .. _Python documentation: https://docs.python.org/
```

Reply:
Oh, that's very much possible, if Python decides to have v4 (not very soon though). Well, anyway, `https://docs.python.org/` redirects to `https://docs.python.org/3/`. I think it (`/3/`) could be omitted.